icu4c/source/common/locid.cpp

```diff
@@ -1030,7 +1030,8 @@ Locale::forLanguageTag(StringPiece tag, UErrorCode& status)
     // uloc_forLanguageTag() to parse it if it somehow is able to detect that
     // the string actually is BCP-47. This works well for things like strings
     // using BCP-47 extensions, but it does not at all work for things like
-    // BCP-47 grandfathered tags (eg. "en-GB-oed") which are possible to also
+    // legacy language tags (marked as “Type: grandfathered” in BCP 47, e.g., "en-GB-oed")
```

Comment:
I'd very much prefer if you used the same quotation marks consistently, either “xxx” or "xxx", but not mixing them, especially not on the same line for no obvious reason. The exising ICU4C source code uses the latter, "ASCII quotes", almost exclusively so it might be a good idea to simply stick with that.

Reply:
Almost all quotes in ICU docs are ASCII quotes because we were limited to ASCII + HTML entity refs until recently.
I would like to use real quotes for human quoting when I think of them, and ASCII quotes when it's more like a code string literal so someone could copy & paste it into real source code.